doc/source/development/release_process.rst

```diff
@@ -159,10 +159,6 @@ Promote Nexus Repository
 * Select the "Releases" repository and click "Promote".
 * Next click on "Repositories", select the "Releases" repository and validate that your artifacts exist as you expect them.
 
-Publish the Bintray Uploaded Distribution Packages
----------------------------------------
-
-Log into bintray and publish the uploaded artifacts.
```

Comment:
I am not familiar with this artifactory but I assume we will need similar step here?

Reply:
No. It has not such "publish" step. What's uploaded to it is immediately available to download.

Reply:
Great, thanks for confirming :-)